src/css/layout/09-footer.css

```diff
@@ -1,2 +1,226 @@
-.footer {
+/* Basic Styles */
+footer {
+    background-color: #121212;
+    color: var(--color-white);
+    padding: 40px 20px 20px;
 }
+
+@media (min-width: 768px) {
+    footer {
+        padding: 32px 32px 0;
+    }
+}
+
+@media (min-width: 1280px) {
+    footer {
+        padding: 64px 128px 0;
+    }
+}
+
+.container {
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+/* Footer Content */
+.footer__content {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-bottom: 48px;
+}
+
+@media (min-width: 767px) {
+    .footer__content {
+        flex-direction: row;
+        justify-content: space-between;
+        margin-bottom: 110px;
+    }
+}
+
+.footer__logo {
+    margin-bottom: 20px;
+}
+
+@media (min-width: 767px) {
+    .footer__logo {
+        margin-bottom: 32px;
+    }
+}
+
+.footer__logo__text {
+    margin: 0;
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 1.36;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+}
+
+.footer__logo-highlight {
+    color: var(--color-green);
+}
+
+.footer__address {
+    margin-bottom: 40px;
+}
+
+.footer__address__list {
+    list-style: none;
+    padding: 0;
+    gap: 16px;
+    display: flex;
+    flex-direction: column;
+}
+
+.footer__address--phone,
+.footer__address--address--text {
+    color: rgba(255, 255, 255, 0.60);
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 1.28;
+    letter-spacing: -0.28px;
+}
+
+.footer__address--phone--link,
+.footer__address--address--text--second-line {
+    font-family: 'Manrope', sans-serif;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.28;
+}
+
+.footer__social--and--links {
+    gap: 48px;
+    display: flex;
+    flex-direction: column;
+}
+
+@media (min-width: 768px) {
+    .footer__social--and--links {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        width: 50%;
+    }
+}
+
+@media (min-width: 1280px) {
+    .footer__social--and--links {
+        gap: 290px;
+    }
+}
+
+.footer__links--list {
+    list-style: none;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.footer__links__list--item {
+    margin-bottom: 10px;
+}
+
+.footer__links--link {
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.28;
+}
+
+.footer__links--link:hover {
+    color: #4caf50;
+}
+
```

Comment:
I think we should use the same styles in terms of animations for these buttons, both for the header and for the footer

Reply:
fixed